### PR TITLE
Added support for Pam's HarvestCraft 2 Core, Crops, Trees, and Extended

### PR DESCRIPTION
**Pull request by silentnyte**
There is an existing error in phc2foodextended with minecraft:honey_bottle tags

### CODE REVIEW

File: settings.gradle [hunk 1]
```diff
@@ -45,3 +45,5 @@ include 'patchwork-resource'
 include 'patchwork-tags'
 include 'patchwork-tooltype'
 include 'patchwork-vanilla-patches'
+include 'patchwork-entity'
```
Comment (famous1622) [inline settings.gradle:48]: These includes should be in alphabetical order

File: patchwork-entity/build.gradle [hunk 2]
```diff
@@ -0,0 +1,10 @@
+archivesBaseName = "patchwork-entity"
+version = getSubprojectVersion(project, "0.1.0")
+
+dependencies {
+	implementation project(path: ':patchwork-api-base', configuration: 'dev')
+}
+
+loom {
+	accessWidener "src/main/resources/patchwork-MobEntity.accesswidener"
```
Comment (famous1622) [inline patchwork-entity/build.gradle:9]: Rename the AW to match the module name

**Review by famous1622 — CHANGES_REQUESTED** [review]
Mostly looks good, just a few small nitpicks

File: patchwork-entity/build.gradle (hunk 2 above)
Comment (silentnyte) [reply patchwork-entity/build.gradle:9]: Oh it was my understanding that the file name should match the class that you are widening.
As for the module name, I asked on discord and patchwork-entity seemed acceptable. There was mention of patchwork-extensions-entity. I'm good with either just let me know.

**Review by TheGlitch76 — CHANGES_REQUESTED** [review]
The access widener itself looks OK, but I'd like it to be moved to `patchwork-extensions-entity` as was apparently discussed on Discord.

File: settings.gradle [hunk 3]
```diff
@@ -47,3 +47,4 @@ include 'patchwork-tags'
 include 'patchwork-tooltype'
 include 'patchwork-vanilla-patches'
 
+
```
Comment (TheGlitch76) [inline settings.gradle:50]:
```suggestion
```

File: patchwork-extensions-entity/src/main/resources/fabric.mod.json [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    "sources": "https://github.com/PatchworkMC/patchwork-api"
+  },
+  "authors": [
+    "PatchworkMC"
+  ],
+  "accessWidener": "patchwork-MobEntity.accesswidener",
+  "depends": {
+    "patchwork-api-base": "*"
+  },
+  "description": "Access Widener for MobEntity.",
```
Comment (TheGlitch76) [inline patchwork-extensions-entity/src/main/resources/fabric.mod.json:19]:
```suggestion
  "description": "Various extensions and APIs related to entities.",
```

**Review by TheGlitch76 — APPROVED** [review]

**Comment by ZNixian** [comment]
I've probably missed quite a bit from discord, but is there are reason why you haven't merged this after approving it?

**Pull request merged**